status/queues.py

```diff
@@ -48,7 +48,7 @@ def get(self):
                     library_type = ''
                     runmode = ''
                     if not 'lambda DNA' in art.name:
-                        library_type = art.samples[0].project.udf["Library construction method"]
+                        library_type = art.samples[0].project.udf.get("Library construction method", 'NA')
                         try:
```

Comment:
I guess you could use get here as well?

Reply:
Maybe next time? The code hasn't really changed here?

Reply:
I was thinking the same as both of you when I adjusted the code according to Anands recommendation. I'll just keep it as it is for now.